src/main/java/ladder/LadderApplication.java

```diff
@@ -14,6 +16,27 @@ public static void main(String[] args) throws Exception {
         ResultView.printLadder(ladderGame, ladderRewards);
 
         LadderGameResult ladderResult = ladderGame.play(ladderRewards);
-        ResultView.showResult(ladderResult);
+        execute(ladderResult);
+    }
+
+    private static void execute(LadderGameResult ladderResult) {
+
+        GameStatus gameStatusStatus = new GameStatus(true);
+        while(gameStatusStatus.repeatable())
+        {
```

Comment:
```suggestion
        while (gameStatusStatus.repeatable()) {
```
기존에 작성하시던 컨벤션과 조금 달라진 것 같아요 🤔

Reply:
컨벤션을 중복하게 작성했네요
gameStatus로 수정하겠습니다